OpenSim/Actuators/ActiveForceLengthCurve.cpp

```diff
@@ -158,6 +158,15 @@ double ActiveForceLengthCurve::calcValue(double normFiberLength) const
     return m_curve.calcValue(normFiberLength);
 }
 
+SmoothSegmentedFunction::ValueAndDerivative ActiveForceLengthCurve::
```

Comment:
Ok for now, but I would recommend a fast refactor after this PR is turned around to move `ValueAndDerivative` into a separate header. This naming is exposing an inner detail.